cmd/pullrequest-init/example/base.json

```diff
@@ -0,0 +1 @@
+{"Ref":"master","Sha":"723b9a9d560bdf4dc8fc6f697d53f662d3454ac8","Repo":{"ID":"146641150","Namespace":"tektoncd","Name":"pipeline","FullName":"tektoncd/pipeline","Perm":{"Pull":false,"Push":false,"Admin":false},"Branch":"master","Private":false,"Clone":"https://github.com/tektoncd/pipeline.git","CloneSSH":"git@github.com:tektoncd/pipeline.git","Link":"https://github.com/tektoncd/pipeline","Created":"2018-08-29T18:21:55Z","Updated":"2019-12-04T17:26:42Z"}}
```

Comment:
I see a lot of JSON files under pullrequest-init/example...is this intentional?

Reply:
Yup! Instead of embedding each payload in the resource docs, I figured it'd be more useful to include a complete example.

Reply:
nit: we could use `./testdata` folder if we want to use those for tests (it's a special folder in the go world)

Reply:
yeah along the same lines, if this is just for documentation purposes, maybe there is a better folder than inside `cmd`. And if its for tests, maybe a `testdata`. But this is a pretty minor thing so that should not hold up this PR!